packages/yoga/src/Input/web/Input.test.jsx

```diff
@@ -204,7 +204,42 @@ describe('<Input />', () => {
         </ThemeProvider>,
       );
 
+      debug();
+
       expect(screen.queryByRole('button')).not.toBeNull();
     });
+
+    it('should have aria-label', () => {
+      const value = 'aria label value';
+      const { debug, getByTestId } = render(
+        <ThemeProvider>
+          <Input label="Input" value="foo" ariaLabel={value} />
+        </ThemeProvider>,
+      );
+
+      debug();
+
+      const inputElement = getByTestId('input');
+
+      expect(inputElement).toBeInTheDocument();
+
+      expect(inputElement).toHaveAttribute('aria-label', value);
+    });
+    it('should have label value', () => {
+      const value = 'label value';
+      const { debug, getByTestId } = render(
+        <ThemeProvider>
+          <Input label={value} value="foo" />
+        </ThemeProvider>,
+      );
+
+      debug();
```

Comment:
❓ Doubt | @nataliosilveiraext 

I think that you forgot this `debug` here.

Reply:
was forgotten, I will remove it